dir.targets

```diff
@@ -81,11 +87,11 @@
          with state.  They were passing because MSTest was serial. -->
 
     <!-- Run tests on the full framework -->
-    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.x86.exe @(MainAssembly, ' ') -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html $(XunitOptions) > %(MainAssembly.FullPath)_stdout.txt"
+    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.exe @(MainAssembly, ' ') $(XunitOptions) -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html > %(MainAssembly.FullPath)_stdout.txt"
```

Comment:
I'll revert this change. First, about 11 extra tests fail with x64 runner. Second, I think if you run under x86 you may want to use x86 runner anyway.